subprojects/build-cache/src/main/resources/h2/schemas/org.gradle.caching.local.internal.H2BuildCacheService.sql

```diff
@@ -22,3 +22,12 @@ CREATE TABLE IF NOT EXISTS filestore.catalog
     entry_size    BIGINT              NOT NULL,
     entry_content BINARY LARGE OBJECT NOT NULL
 );
+
+CREATE TABLE IF NOT EXISTS filestore.lru
+(
+    entry_key     VARCHAR(32)         NOT NULL PRIMARY KEY,
+    entry_accessed    BIGINT          NOT NULL
+);
```

Comment:
Just so we have documented also here in PR:
I tested a single table solution with this change: https://github.com/gradle/gradle/commit/bf7f0e1a06d52d5e81dade8a6741f458529ccdf5.

And it seems single table solution is about 8.5% slower for clean build: 
top is a solution from this PR and bottom is with changes from  https://github.com/gradle/gradle/commit/bf7f0e1a06d52d5e81dade8a6741f458529ccdf5.
<img width="1999" alt="Screenshot 2023-05-24 at 12 17 35" src="https://github.com/gradle/gradle/assets/3939893/7e5839bb-a0a4-4ad9-9d8d-8ac14036311e">

While performance is not that important right now, I think we can still go with faster solution since it doesn't add that much complexity.

Reply:
👍 thanks for mentioning it here, too, for future reference.